docs/guide/contributing.rst

```diff
@@ -31,37 +31,35 @@ We especially need contributions in these areas:
 * Developer tools or docs-as-code toolchains and workflows.
 * High-level discussions of common tools. 
 
-For help finding where a new set of topics best fit into the rest of 
+For help finding the best fit for new topics in the rest of 
 the guide, `ask in Slack`_, or `file a GitHub issue`_.
 
-Guidelines
------------
+Contribution guidelines
+-------------------------
 
-Follow these guidelines for discussing tools: 
+General guidelines include:
 
-* Ensure your content is of interest to a general audience.
-* Avoid advocacy and plugs about your favorite toolchain, even if it's open source.
-* Present specific use cases, how problems were solved, and what worked or didn't work well. 
-* Consider to first `file a GitHub issue`_ or contact one of the guide editors at guide@writethedocs.org.
+* Use a friendly and encouraging tone. It's a good way to write docs, so practice with a guide at Write the Docs!
```

Comment:
```suggestion
* Use a friendly and encouraging tone. It's a good way to write docs, so practice with docs about docs!
```
I think the original was a nice turn of phrase. Not a requirement or anything.

Reply:
That's strange. I honestly have no idea where that "docs about docs" thing came from. I don't recall writing it. It may have been a copy/paste from somewhere else in the doc because there was a lot of repeat content. I'll definitely investigate further.

Reply:
Seems like it's [been there for 6 years](https://github.com/writethedocs/www/blame/32ef981f5490c5dc99c68afc15203eea3712d100/docs/guide/contributing.rst#L44). 🤷

Reply:
Oh, thanks for taking a look. I had it backwards...I thought you were saying I had added the "docs about docs" line. All good now!